jobs/JGLOBAL_WAVE_INIT

```diff
@@ -10,9 +10,10 @@ export errchk=${errchk:-err_chk}
 export MP_PULSE=0
 
 # Set COM Paths
-YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_WAVE_PREP
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
+   COMOUT_WAVE_PREP:COM_WAVE_PREP_TMPL
 
-mkdir -m 775 -p ${COM_WAVE_PREP}
+if [[ ! -d "${COMOUT_WAVE_PREP}" ]]; then mkdir -p "${COMOUT_WAVE_PREP}"; fi
```

Comment:
Some places it is `mkdir -p`, some `mkdir -m 775 -p`.  Which is correct/accurate?

Reply:
I made all of these `mkdir -m 775 -p`. This does the following:

 - The owner of the directory has read, write, and execute permissions (rwx);
- The group associated with the directory has read, write, and execute permissions (rwx);
- All others have read and execute permissions (r-x).

This should not introduce any downstream issues.

Reply:
I am not sure of the origins of `-m 775` with the `mkdir` command.  I don't think `group` should have `write` access to the directory.
I was going to suggest removing all occurrences of `-m 775` or `-m 755` from anywhere it is encountered.
At least, let us not introduce any more in this PR.

Reply:
Reverted/updated.